pandas/tests/util/test_show_versions.py

```diff
@@ -26,13 +33,56 @@
     "ignore:Distutils:UserWarning"
 )
 @pytest.mark.filterwarnings("ignore:Setuptools is replacing distutils:UserWarning")
-def test_show_versions(capsys):
+def test_show_versions(tmpdir):
+    as_json = os.path.join(tmpdir, "test_output.json")
+
+    pd.show_versions(as_json=as_json)
+
+    # make sure that the file was created
+    assert os.path.exists(as_json)
```

Comment:
the next lines will fail if this isn't true, so this strikes me as redundant, I think it can be removed